data/json/effects.json

```diff
@@ -1399,6 +1411,12 @@
             "stim_min_val": [-1]
         }
     },
+    {
+        "type": "effect_type",
+        "id": "anticonvulsant_visible",
```

Comment:
You should probably rename this effect (and antibiotics effects below) to `took_anticonvulsant_visible` to match the `took_prozac_visible` and `took_xanax_visible` effects. Or vice versa, remove `took` part from prozac and xanax.